packages/resolve-scripts/src/template/client/reducers/todos.js

```diff
@@ -1,6 +1,7 @@
 import { createReducer } from 'resolve-redux';
-import todosProjection from '../../common/read-models/todos';
+import readModel from '../../common/read-model';
 
-const { name, projection } = todosProjection;
```

Comment:
Since read-model is likely one for instance, it have no more own name. Every entities are queried by Graphql. So, that name is custom